js/tic_tac_toe.js

```diff
@@ -1,2 +1,69 @@
-// Your code here
-// Use the jQuery
+$(document).ready(function(){
+  $winner = $('#winning');
+  $tiles = $('td');
+  $tiles.map(function(){
```

Comment:
I learned a thing, and wasn't thinking myself that you could use .map in this way!